Classes/Controller/ReceiverController.php

```diff
@@ -195,6 +200,56 @@ protected function deleteAction(string $table, int $uid)
         }
     }
 
+    /**
+     * Create a new record.
+     *
+     * Submitted info is based on a query string as described at
+     * https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Examples/EditLinks/Index.html
+     *
+     * @param array $edit A an array as parsed from `tt_content[-1]=new`
+     * @param array $defVals Default content as an array parsed from `tt_content[title]=title`
+     * @param int $pid Page ID
+     */
+    protected function newAction(array $edit, array $defVals, int $pid)
+    {
+        /** @var DataHandler $dataHandler */
+        $dataHandler = GeneralUtility::makeInstance(DataHandler::class);
+        $dataHandler->BE_USER = $GLOBALS['BE_USER'];
+
+        $table = array_key_first($edit);
```

Comment:
`array_key_first` 
this will break compatibility with php =< 7.3

Reply:
`if (!function_exists('array_key_first')) {
    function array_key_first(array $arr) {
        foreach($arr as $key => $unused) {
            return $key;
        }
        return NULL;
    }
}`

possible solution from PHP documentation